css_tasks/task11/index.html

```diff
@@ -0,0 +1,896 @@
+    <section class="standard container">
+      <header class="main-header">
+        <h2 class="main-header__headline">What we do</h2>
+        <p class="main-header__title">story about us</p>
+        <div class="main-header__separator"></div>
+        <p class="main-header__text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat.
+        </p>
+      </header>
+      <div class="standard__wrapper">
+        <div class="standard__element">
+          <div class="standard__image">
+            <picture>
+              <source
+                media="(max-width:1024px)"
+                srcset="assets/layer-44@2x.jpg"
+              />
+              <img src="assets/layer-44@3x.jpg" alt="Laptop" />
+            </picture>
+            <div class="standard__hover">
+              <picture>
+                <source
+                  media="(max-width:1024px)"
+                  srcset="assets/users@2x.png"
+                />
+                <img src="assets/users@3x.png" alt="" class="hover-image" />
+              </picture>
+            </div>
+          </div>
+        </div>
+        <div class="standard__element">
+          <div class="standard__image">
+            <picture>
+              <source
+                media="(max-width:1024px)"
+                srcset="assets/layer-39@2x.jpg"
+              />
+              <img src="assets/layer-39@3x.jpg" alt="Desk" />
+            </picture>
+            <div class="standard__hover">
+              <picture>
+                <source
+                  media="(max-width:1024px)"
+                  srcset="assets/users@2x.png"
+                />
+                <img src="assets/users@3x.png" alt="" class="hover-image" />
+              </picture>
+            </div>
+          </div>
+        </div>
+        <div class="standard__element">
+          <div class="standard__image">
+            <picture>
+              <source
+                media="(max-width:1024px)"
+                srcset="assets/layer-41@2x.jpg"
+              />
+              <img src="assets/layer-41@3x.jpg" alt="Flower" />
+            </picture>
+            <div class="standard__hover">
+              <picture>
+                <source
+                  media="(max-width:1024px)"
+                  srcset="assets/users@2x.png"
+                />
+                <img src="assets/users@3x.png" alt="" class="hover-image" />
+              </picture>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- Green Bar Decoration -->
+    <div class="decoration decoration-green">
+      <div class="container decoration__wrapper">
+        <div class="decoration__element">
+          <span>42</span>
+          <p>Web Design <br />Projects</p>
```

Comment:
it is more about size of block and fonts, better without br, and generally there is not a lot use cases when you need to use it

for example here we have content from cms and no one will add tags there, it should be just content

Reply:
I got rid of <br /> tags